test/tasks/ods-build-java_test.go

```diff
@@ -0,0 +1,89 @@
+package tasks
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/opendevstack/pipeline/internal/command"
+	"github.com/opendevstack/pipeline/pkg/tasktesting"
+)
+
+func TestTaskODSBuildJava(t *testing.T) {
+	runTaskTestCases(t,
+		"ods-build-java-v0-1-0",
```

Comment:
With the latest changes in `master`, we decided to exclude the version name from the test, use `ods-build-java` as the test case name instead.

Reply:
great